src/structures/Channel.js

```diff
@@ -74,6 +74,15 @@ class Channel extends Base {
     return new Date(this.createdTimestamp);
   }
 
+  /**
+   * Whether this Channel is a partial
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return this instanceof DMChannel && typeof this.lastMessageId === 'undefined';
```

Comment:
This getter could just always return false and have dm channel (the only one that can be partial currently), do this check:
```suggestion
    return false;
```

Reply:
Should I add in the jsdoc a note to say that this is always false if it's not a DMChannel?